docs/documentation/actions/micrometers.md

```diff
@@ -0,0 +1,285 @@
+# Counter
```

Comment:
I would precise that for all actions, we dont need to provide registry as jvm global registry is used by defalut